docs/pages/_meta.json

```diff
@@ -7,6 +7,12 @@
   "applications": {
     "title": "Applications"
   },
+  "use-case1": {
```

Comment:
@darlisagc @johnnygreeney 

Should we create a top-level menu called 'Use Cases' at the same level as 'Applications' and then add a subfolder for each use case (use-case1, use-case2, etc.)?

Reply:
![Screenshot 2024-09-03 at 13 34 25](https://github.com/user-attachments/assets/6ce1a0da-a3d6-4f9d-9543-31e1d6f99bb0)

i was thinking more like this, as otherwise would need 3 levels of subfolders.

Reply:
![Screenshot 2024-09-03 at 13 57 49](https://github.com/user-attachments/assets/71727a20-8eb1-4afe-9d3b-6fa6fdd6d20e)

..alternative was to have all 12 queries on one page, but then it was a little too long.